Something went wrong with: https://www.zillow.com/homedetails/9245-W-Irving-Park-Rd-GE-Schiller-Park-IL-60176/2077842061_zpid/Traceback (most recent call last):
  File "/Users/jarora/PycharmProjects/websiteScarper/zillow/zillow_scarper.py", line 49, in <module>
    zillow.zillow_functions.record(property_link,save_xml=False)
  File "/Users/jarora/PycharmProjects/websiteScarper/zillow/zillow_functions.py", line 234, in record
    dom = record_xml_util(property_page_soup)
  File "/Users/jarora/PycharmProjects/websiteScarper/zillow/zillow_functions.py", line 220, in record_xml_util
    record_xml['hoa_and_financial_details'] = get_hoa_and_financial_details(property_page_soup)
  File "/Users/jarora/PycharmProjects/websiteScarper/zillow/zillow_functions.py", line 176, in get_hoa_and_financial_details
    bold_heading_groups = details_soup.find_all('div', recursive=False)[0].find_all('div', recursive=False)
IndexError: list index out of range
